calico-enterprise/observability/elastic/flow/aggregation.mdx

```diff
@@ -88,6 +86,7 @@ Before [changing the default aggregation level](../../../reference/resources/fel
 
 - Although any change in aggregation level affects flow log volume, lowering the aggregation number (especially to `0` for no aggregation) will cause significant impacts to log storage. If you allow more flow logs, ensure that you provision more log storage.
 - Verify that the parameters that you want to see in your aggregation level are not already [filtered](filtering.mdx).
+- Note that currently the numeric levels must be used instead of the names in configuration.
```

Comment:
One suggestion is to move this not up to the top. Right after `For allowed flows, the default aggregation level is 2, AggregateIPsAndSourcePorts and for denied flows the default aggregation level is 1,
AggregateSourcePorts.`